### DISCUSSION TITLE
What are the roles and functions of task_broker.py and task_executor.py files?

### DISCUSSION THREAD

**Discussion by DAAworld** [infiniflow/ragflow#1542] [Q&A]
I don't understand what these two files do. I have tried to use only the document recognition, splitting, and merging functions of ragflow without esearch, but it failed. Can you explain their functions to me?

**Comment by KevinHuSh** [comment]
task_broker.py is deprecated.
task_executor.py is for executing task of parsing documents.
Please refer to rag/app/*.py which are all about parsing document.

**Reply by DAAworld** [reply]
OK,think you.
